events/wmt22.md

```diff
@@ -125,11 +125,6 @@ Confirmed language pairs:
 > System papers must describe one or more shared task submissions. System paper submissions that we cannot link to a shared task submission will be rejected without review. System papers can overlap with other published work, and do not have to follow the double submission policy. There is no maximum length for system papers, but normally a short paper (4-6 pages) is appropriate. System papers should not be anonymised.
 
 
-## Poster format
-
-Posters must be presented in Gather.Town.
```

Comment:
Have done the same here, as this one doesn't even follow the rules in the [website](https://www.statmt.org/wmt22/#:~:text=PRESENTATION%20AND%20POSTER%20FORMAT&text=For%20the%20in%2Dperson%20poster,that%20are%20portrait%20in%20orientation.).

Reply:
Yeah, that seems to changed from the time I included it. Thanks!